app/models/config.py

```diff
@@ -88,6 +112,33 @@ def get_attrs(self):
                 if not name.startswith("__")
                 and (type(attr) is bool or type(attr) is str)}
 
+    @property
+    def style(self) -> str:
+        """Returns the default style updated with specified modifications.
+
+        Returns:
+            str -- the new style
+        """
+        style_sheet = cssutils.parseString(
+            open(os.path.join(current_app.config['STATIC_FOLDER'],
+                              'css/variables.css')).read()
+        )
+
+        modified_sheet = cssutils.parseString(self.style_modified)
+        for rule in modified_sheet:
+            rule_default = get_rule_for_selector(style_sheet,
+                                                 rule.selectorText)
+            # if modified rule is in default stylesheet, update it
+            if rule_default is not None:
+                # TODO: update this in a smarter way to handle :root better
+                # for now if we change a varialbe in :root all other default
+                # variables need to be also present
```

Comment:
The variables in `:root` can still be overridden with the html tag though, for example:

```css
html {
    --whoogle-page-bg: #f0f0f0 !important;
}
```

It might be good enough to just update the CSS wiki page to indicate that individual variables can be overridden that way.